packages/language-server/src/plugins/svelte/SvelteDocument.ts

```diff
@@ -208,11 +210,20 @@ export class SvelteFragmentMapper {
             );
         }
 
-        return new SvelteFragmentMapper(
-            { end: -1, diff: 0 },
-            new IdentityMapper(originalDoc.uri),
-            new IdentityMapper(originalDoc.uri),
-            sourceMapper,
+        return null;
+    }
+
+    private static async createSourceMapper(processed: Processed[], originalDoc: Document) {
+        return await processed.reduce(
```

Comment:
`return await` in an `async` method is usually redundant

```suggestion
        return processed.reduce(
```